### PR TITLE
Change rdesktop (deprecated) to xfreedesktop.

### PR DESCRIPTION
**Pull request by irawoodring**
rdesktop is deprecated; we now use xfreedesktop which is current (and maintained).

### CODE REVIEW

File: manual/winserv.rst [hunk 1]
```diff
@@ -55,11 +55,11 @@ GNU/Linux
 
 GNU/Linux systems usually have the rdesktop_ command-line tool in their repositories. If not, it can most likely be built from source. rdesktop provides a geometry flag that accepts both screen percentages or resolutions from the command-line to help adjust the client to an appropriate size. For instance, to allow the client to take up 90% of your screen::
```
Comment (Liluye) [inline manual/winserv.rst]: If rdesktop isn't being used any more, you may not want to reference it in the paragraph.

**Comment by micklautenbach** [comment]
:+1:

**Comment by Liluye** [comment]
:+1:

**Comment by seanfisk** [comment]
:+1:, but you'll have to merge in `master` and resolve conflicts before this can be merged through the GitHub interface.

**Pull request merged**